pkg/commands/version/get_command.go

```diff
@@ -16,20 +16,22 @@
 package version
 
 import (
-	"fmt"
-
+	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/commands/cmdlib"
 	"github.com/GoogleCloudPlatform/k8s-cluster-bundle/pkg/version"
 	"github.com/spf13/cobra"
 )
 
 // GetCommand returns the version command.
-func GetCommand() *cobra.Command {
+func GetCommand(cio *cmdlib.CmdIO) *cobra.Command {
```

Comment:
I just want to point this out, and I'm not blocking you on this, I just  would personally avoid doing this, and it's a good simple low impact example of a problem I often see:

I'll explain:
By doing this, you are now making the version package depend upon cmdlib. 
Cmdlib target depends on 
~~~~
        "//pkg/apis/bundle/v1alpha1:go_default_library",
        "//pkg/build:go_default_library",
        "//pkg/converter:go_default_library",
        "//pkg/files:go_default_library",
        "//pkg/options:go_default_library",
        "//pkg/wrapper:go_default_library",
~~~~

This isn't ideal.  This couples the code together.

Instead, if you want this same behavior, and you want to use this,  have getcommand take interface that are defined in the version package, and bind the implementation elsewhere. 
For example, you can easily, easily define a writeTo via an arg writeTo(stuffToWrite string), or an onWriteComplete()  local to this module.

You can then bind it in cmdlib. 

In general I encourage you to not directly take dependencies on code that you do not need to.  
You end up making this piece of code bound to the implementation of other modules (because it directly shares code with them)

Reply:
I could move cmdlib into `cmdio` and `bundleio` to break it up. Would that help from your perspective?